techniques/system/common/1.0/rudder_directives.st

```diff
@@ -0,0 +1,28 @@
+#####################################################################################
+# Copyright 2013 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+# This file is the main entry points for the bundle sequence for 
+# Rudder directives. It is actually a list of methods, and not
+# a bundle sequence, which allows to add the technique or directives 
+# names as promisee. 
```

Comment:
s/promisee/promisers/

Promisee != promiser. See http://syslog.me/2013/05/21/the-value-of-the-promisee-in-cfengine-promises/.